python/pyspark/mllib/classification.py

```diff
@@ -102,6 +102,7 @@ class LogisticRegressionModel(LinearClassificationModel):
       in Multinomial Logistic Regression. By default, it is binary
       logistic regression so numClasses will be set to 2.
 
+    >>> from pyspark.mllib.linalg import SparseVector
```

Comment:
I don't think PySpark examples import relevant PySpark classes within doctests.

Reply:
The examples are actually ran in the tests. If the SparseVector is imported in the Python file itself, there is no issue. However, if the import isn't used outside of the example, I would suggest to move them inside of the example. An error:
```
**********************************************************************
File "/home/runner/work/spark/spark/python/pyspark/mllib/feature.py", line 287, in __main__.ChiSqSelector
Failed example:
    data = sc.parallelize([
        LabeledPoint(0.0, SparseVector(3, {0: 8.0, 1: 7.0})),
        LabeledPoint(1.0, SparseVector(3, {1: 9.0, 2: 6.0})),
        LabeledPoint(1.0, [0.0, 9.0, 8.0]),
        LabeledPoint(2.0, [7.0, 9.0, 5.0]),
        LabeledPoint(2.0, [8.0, 7.0, 3.0])
    ])
Exception raised:
    Traceback (most recent call last):
      File "/opt/hostedtoolcache/Python/3.6.11/x64/lib/python3.6/doctest.py", line 1330, in __run
        compileflags, 1), test.globs)
      File "<doctest __main__.ChiSqSelector[0]>", line 2, in <module>
        LabeledPoint(0.0, SparseVector(3, {0: 8.0, 1: 7.0})),
    NameError: name 'LabeledPoint' is not defined
```